scripts/add-cluster.sh

```diff
@@ -260,6 +260,12 @@ while test $# -gt 0; do
 done
 
 CLUSTER_JOIN_TO="host"
+# Since MULTI_MEMBER variable is appended at the end of kubernetes object name for toolchaincluster resource,
+# let's always set a "member id" if not provided, so that we are sure that those object names will end with an alphanumerical char.
+if [ -z "$MULTI_MEMBER" ]
+then
+      MULTI_MEMBER=1
+fi
```

Comment:
Why do we need it? 
One complication with this is that it won't be compatible with already existing clusters. When we need to update the (Cluster)Roles that are granted via this script, then we just re-run it, so we need to keep it compatible.

Reply:
I see,

but we need it because , this way we are sure also member1 toolchaincluster name ends with and alphanumerical char always (in this case a "1"), otherwise the truncate logic may leave an invalid char at the end.

Reply:
I can maybe add it only if the last char is not alphanumerical ? 🤔

Reply:
either that or add it only when  the length of the `JOINING_CLUSTER_NAME` needs to be truncated

Reply:
I was easier to add it only if `MULTI_MEMBER` was not provided already and if JOINING_CLUSTER_NAME was > 63 chars.

please see: [3f65aba](https://github.com/codeready-toolchain/toolchain-cicd/pull/83/commits/3f65abad4930985626c0c2638a034ca28284288a)